src/app/client/src/app/modules/public/module/explore/components/explore-content/explore-content.component.ts

```diff
@@ -39,6 +39,7 @@ export class ExploreContentComponent implements OnInit, OnDestroy, AfterViewInit
   public contentList: Array<ICard> = [];
   public cardIntractEdata: IInteractEventEdata;
   public loaderMessage: ILoaderMessage;
+  public numberOfSections = 20;
```

Comment:
can you use public numberOfContents = new Array(this.configService.appConfig.SEARCH.PAGE_LIMIT);
same for explore section page also and for sections create appConfig.Section.LIMIT

Reply:
Sure.